### ISSUE TITLE
Add Stop button

### ISSUE THREAD

**Issue by Balta92** [OMGSoundboard/android-app#126]
## :warning: Is your feature request related to a problem? Please describe
I have some long sounds and songs that I want to play briefly depending on the context, they can become annoying because the file have a long time duration

## :bulb: Describe the solution you'd like
Other soundboards have a stop button, this can solve the problem but this isnt available

## :bar_chart: Describe alternatives you've considered
I can mute multimedia audio volume but its not ideal as this take time in wear os and can lead to annoyances with other apps and general usage

## :page_facing_up: Additional context
The wear OS app should receive this feature too.

## :raising_hand: Do you want to develop this feature yourself?

- [ ] Yes
- [x] No (I dont have the necessary knowledge)

**Comment by OMGSoundboard** [comment]
Thanks for your feedback, @Balta92.

I am not sure if I understand your usecase correctly. How long are your tunes? Would you need some kind of fast forward and rewind options too?

**Comment by Balta92** [comment]
Some of my files have a duration of 10 seconds, others can extend up to 30 seconds. But even for a 5 seconds file I often stop it after 2 or 3 seconds to avoid becoming annoying, depending on the context. For me a fast forward or rewind buttons are not necessary, just a stop button.

**Comment by jbprojectstudio** [comment]
I agree with the request; it would be enough if you clicked it again and it stopped. It would be very helpful if when you change the sound, they didn't overlap, but the previous one would stop (perhaps as an option).

**Comment by jbprojectstudio** [comment]
I agree with the request; it would be enough if you clicked it again and it stopped. It would be very helpful if when you change the sound, they didn't overlap, but the previous one would stop (perhaps as an option).

**Comment by qwertfisch** [comment]
The [MemeBoard](https://f-droid.org/de/packages/com.bluefirestudios.memeboard/) app shows a stop button overlay as long as a sample is playing. Also: selecting another sample stops the current sample without blending/overlapping.

These features would be great. Maybe the stop button can be put in the top bar? I know, there are already the favorites and search buttons, but I consider them not as important as stopping a sound immediately.

For overlapping sounds, this could be a setting, so that users can choose their preference.
